docs/community/tutorials-contest/2022/local-subnet-development/README.md

```diff
@@ -0,0 +1,358 @@
+# Set up your development environment for local subnet development
```

Comment:
All section titles should be title cases. 

All word `Subnet` should have letter capitalized.